docs/network/smart-rollup-nodes.md

```diff
@@ -185,30 +186,21 @@ Follow these steps to convert a Smart Rollup node from observer mode to maintena
 
 1. Stop the node.
 
-1. Re-initialize the node for maintenance mode by running the `init maintenance config` command and passing the addresses or Octez aliases of the accounts.
+1. Run the node in maintenance mode, passing the addresses or Octez aliases of the accounts and the layer 1 node that you control.
 This example uses `$OPERATOR_ACCOUNT` for the account with 10,000 liquid tez and `$SECONDARY_ACCOUNT` for the other account:
 
    ```bash
-   octez-smart-rollup-node init maintenance config for sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf \
+   octez-smart-rollup-node run maintenance for sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf \
      with operators \
      operating:$OPERATOR_ACCOUNT \
      cementing:$SECONDARY_ACCOUNT \
      executing_outbox:$SECONDARY_ACCOUNT \
```

Comment:
If you don't prefix the key with the mode the key will be used as default for all not set mode:

```
octez-smart-rollup-node run maintenance for sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf \
     with operators $OPERATOR_ACCOUNT  [...]
 ```
 This will uses `$OPERATOR_ACCOUNT` for all possible operation to inject (publish, refute and cementing here)
 
 Or 
 ```
 octez-smart-rollup-node run maintenance for sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf \
     with operators $DEFAULT_ACCOUNT operating:$OPERATOR_ACCOUNT  [...]
 ```
This will uses the `$OPERATOR_ACCOUNT` for publishing and refuting, but `$DEFAULT_ACCOUNT` for the rest (here cementing)

In you cc, you must set `executing_outbox` but it won't execute any because it's not configured for, be a private rollup. I'm not sure but maybe there is a config option to execute all outbox messages, I forgot if it was developped or not yet.

This is more information for you, but not necessarely to be reflected on the doc ? Wdyt?

Reply:
That was what I was trying to get at with:

> The first address after `with operators` is the default address that the node uses for operations. You can use different accounts for specific operations by adding the operation and the address to the command. This example uses...

Hopefully that's clear.